sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -916,10 +951,29 @@ export interface AzureMySqlTableDataset extends Dataset {
 
 // @public
 export interface AzurePostgreSqlLinkedService extends LinkedService {
+    azureCloudType?: any;
+    commandTimeout?: any;
     connectionString?: any;
+    credential?: CredentialReference;
+    database?: any;
+    encoding?: any;
     encryptedCredential?: any;
     password?: AzureKeyVaultSecretReference;
+    port?: any;
+    readBufferSize?: any;
+    server?: any;
+    servicePrincipalCredentialType?: any;
+    servicePrincipalEmbeddedCert?: SecretBaseUnion;
+    servicePrincipalEmbeddedCertPassword?: SecretBaseUnion;
+    servicePrincipalId?: any;
+    servicePrincipalKey?: SecretBaseUnion;
+    sslMode?: any;
+    tenant?: any;
+    timeout?: any;
+    timezone?: any;
+    trustServerCertificate?: any;
     type: "AzurePostgreSql";
+    username?: any;
```

Comment:
interesting, is this expected? why so many `any` types here?

Reply:
https://github.com/Azure/azure-rest-api-specs/commit/1ebdf037eccbb271e6c65bffe63e26b7bb17aec3#diff-32c4234669508ebda827325c9d0ba05687b4825bcacf73c40482c20ce04f02f8

The api add new app model properties for Sql server family connectors.

Reply:
Got it and thanks!